src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,115 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 16;
+    static final float LOAD_FACTOR = 0.75f;
+
+    private int capasity;
```

Comment:
Please, correct the spelling of the field 'capacity'. It's important to name variables and methods correctly to improve code readability and to avoid confusion.